backend/src/main/java/ch/puzzle/pcts/dto/member/MemberDto.java

```diff
@@ -0,0 +1,17 @@
+package ch.puzzle.pcts.dto.member;
+
+import ch.puzzle.pcts.model.member.EmploymentState;
+import ch.puzzle.pcts.model.organisation_unit.OrganisationUnit;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.Date;
+
+public record MemberDto(
```

Comment:
are we sure we want the validation on the dto? this would require to manage them twice otherwise you wont have them on the db. I suggest something like in the okr where we invoke the validation rules from the entity do enforce on the dto. Also a DTO theroetically is allowed to be invalid depending on the context. For example here, it is valid to have no `id` set when its a post. @MasterEvarior  please evaluate what you like more dto validation or entity validation.